Makefile

```diff
@@ -8,7 +8,7 @@ IMAGE_BUILD_CMD ?= docker build
 IMAGE_BUILD_EXTRA_OPTS ?=
 IMAGE_PUSH_CMD ?= docker push
 CONTAINER_RUN_CMD ?= docker run
-BASE_IMAGE_FULL ?= debian:bullseye-slim
+BASE_IMAGE_FULL ?= debian:bullseye
```

Comment:
Why change from `debian:bullseye-slim` to `debian:bullseye`, won't it increase the image size? Do we need any additional goodies now that the slim image did not have?

Reply:
yes, I have updated the code to use debian:bullseye-slim itself.. had converted the PR to draft to make that change